fvtest/compilertest/README.md

```diff
@@ -7,11 +7,13 @@ language independent compiler tests for the Testarossa compiler technology.
 Building
 --------
 
-To build the Test compiler, typing `make` in this directory
-(`omr/fvtest/compilertest`) should suffice. A binary called 
-`testjit` will be produced in `../../objs/compilertest_$(BUILD_CONFIG)/`
 
-BUILD_CONFIG defaults to prod. 
+After running `./configure --enable-fvtest --enable-OMR_JIT,`To build the Test
```

Comment:
Fix the punctuation in this sentence to make it more readable.  Remove the comma from within the quotes, and make "To build" lower case.